.gitignore

```diff
@@ -52,3 +52,6 @@ benchmarks/results
 .idea
 .vscode
 *.lock
+
+# add mdanalysis dir 
+mdanalysis/
```

Comment:
Why was this added? If this is only specific to your setup, please remove it.

Please also remove all the files (`.dll`, `.exe`, ...) that have been added to this PR from such folder.

Reply:
I removed the environment files, sorry about that.